sentinel/src/transitions/update_notifications.js

```diff
@@ -1,14 +1,14 @@
-var async = require('async'),
-  _ = require('underscore'),
+var _ = require('underscore'),
   config = require('../config'),
   utils = require('../lib/utils'),
   messages = require('../lib/messages'),
   validation = require('../lib/validation'),
-  db = require('../db-pouch'),
   transitionUtils = require('./utils'),
-  NAME = 'update_notifications';
+  NAME = 'update_notifications',
+  mutingUtils = require('../lib/muting_utils'),
+  logger = require('../lib/logger');
 
-var isConfigured = function(config, eventType) {
+/*var isConfigured = function(config, eventType) {
```

Comment:
Looks like you've left some obsolete code in here?

Reply:
I was sure I had removed that :( Sorry. Thanks for the review!